crates/xtask/src/changelog.rs

```diff
@@ -29,6 +30,16 @@ pub enum ReleaseType {
     Patch,
 }
 
+impl Display for ReleaseType {
```

Comment:
Unrelated but I think it's worth doing it soon, is to rename `ReleaseType` to `Scope` to be a bit more abstract, because this type is used for 2 things:
- Describe the scope or impact of a change.
- Describe the highest scope of a change between 2 releases.

And the second thing is actually only a consequence of the first, so the name should be more relevant to the first, which is describing the scope/impact of a change.

Reply:
Actually, we could also use the name `Severity` since it's used [here](https://doc.rust-lang.org/cargo/reference/semver.html).